src/components/HowItWorks/HowItWorks.scss

```diff
@@ -61,30 +61,11 @@ section.how-it-works-mobile {
       border-top-right-radius: 150px;
     }
   }
-  .support-search {
-    height: 500px;
-  }
-  .navbar-expand-md {
-    .navbar-nav {
-      .nav-link {
-        padding-right: 1rem;
-        padding-left: 1rem;
-      }
-    }
-  }
-  nav.navbar {
-    padding: 20px 60px;
-  }
 }
 @media (max-width: 768px) {
   &:root {
     font-size: 16px;
```

Comment:
Hmm, this looks fishy. It looks like a App.scss media query not something that the howitworks module require. PLease take a look :)